docs/dashboards/chart-builder.md

```diff
@@ -0,0 +1,176 @@
+from the report level.
+
+### Aggregation functions
+
+Back to the example from the section above, if you choose aggregation function instead of `all separately`
+only one data series is created for all the hosts. The result depends on the aggregation function
+function you choose. The aggregation function determines how the metrics are
+aggregated into a single data series. You can pick one of `avg`, `sum`, `min`, `max`
+
+
+## Rollup by
+
+Datapoints sent to Sematext have time label. If a metric is updated every 10 second, and you are
+looking at 24 hours time range, you need 14400 points to display all datapoints. 
+Presenting so many datapoints on the chart would make the chart not readable and would impact
+the web browser performance. If we build time buckets each containing 600 datapoints
+we will have 24 datapoints to present on the chart (each datapoint for each hour). Rollup function
+decides how time buckets are built (what aggregation is performed).
+
+Each of predefined metrics has its own default rollup function defined depending on the type of
+a metric. All the gauge metrics use average rollup function and all metrics of a type counter
+use sum rollup function. In some cases it makes sense to customize the setting.
+
+_NOTE: Recommended for expert users._
+
+
+## Variables
+
+Variables are used to dynamically filter and group by one or more widgets in the report by tags.
+
+Filters from the `Filter by` section don't have to be configured statically when a chart
+is created. Sometimes you need charts on the dashboard to react on a global filter
+change. Consider a case where you're troubleshooting a system and you realized that one of the
+hosts has problems. You have a dashboard which contains aggregate information for all hosts. You can
+of course reconfigure all the components from the dashboard so they show only interesting host data.
+As you might expect this is not a very efficient way of troubleshooting. Here is where Variables
+become a very useful feature. You can just create a tag variable which points to field `os.host`.
+Later this variable should be added to the charts which should listen for variable changes.
+A variable is automatically added on top of the dashboard. When you change the tag values in the
+variable charts will automatically reload the content to match the filter.
+
+Variables can be used not only in `Filter by` section but also in group by section. Suppose you
+have a dashboard with a couple charts which show aggregated metrics about disk usage on many hosts.
+You want to troubleshoot a problem that you spotted on one of the charts (small jump on disk usage).
+To identify the root cause you can try to find the problem by grouping metrics by `os.host`
+tag. A group variable can let you to do that from the level of the dashboard without changing the
+definition of series on any charts. Now imagine that you grouped data by `os.host` but still don't
```

Comment:
definition of series on any charts. Now imagine that you grouped data by `os.host` but still don't
=>
definition of the series in any of the charts. Now imagine that you grouped data by `os.host` but still don't